### PR TITLE
Enhancement: Add text alignment options to verse block toolbar

### PR DESCRIPTION
**Pull request by nfmohit**
## Description
This PR addresses #6695 which requested a text alignment option (left/centre/right) in the verse block as we can see in the heading/paragraph/subhead and other blocks.

## How has this been tested?
This PR has been tested by adding a "Verse" block in Gutenberg editor and locating the alignment settings of the block in the block toolbar. It was also made sure that the alignment settings were correctly saved and rendered in the front-end. This was tested in WP 4.9.5, Gutenberg 2.8.0, Apache server with PHP 7.2.0 and MySQL 5.6.34. According to initial tests, the code doesn’t seem to affect any other areas.

## Screenshots 
gutenberg-6695-min

## Types of changes
This PR imports the dependencies required for the verse block toolbar and text alignment options ( i.e. `Fragment`, `BlockControls` and `AlignmentToolbar` ), adds the `align` attributes in the `edit` and `save` functions and lastly outputs the block toolbar.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation.

### CODE REVIEW

File: core-blocks/verse/index.js [hunk 1]
```diff
@@ -29,6 +34,9 @@ export const settings = {
 			source: 'children',
 			selector: 'pre',
 		},
+		align: {
```
Comment (jorgefilipecosta) [inline core-blocks/verse/index.js]: Hi @nfmohit, thank you for your contribution!
The changes look good from the code point of view.
I'm not sure if other people agree with this thought, but in my option, this attribute should be named textAlign. Align attribute is used by the block align hook (supports: align) and maybe in the future we want to add block alignment here so using textAlign attribute we would have the door open to that :)

Comment (nfmohit) [reply core-blocks/verse/index.js]: Thank you for reviewing this @jorgefilipecosta! I've added [a new commit](https://github.com/WordPress/gutenberg/pull/6700/commits/bb887bcda43780254c7b86eacaa1e880d3ad6bfb) which changes `align` to `textAlign`.

I can see the other core blocks like header, subhead, paragraph etc. blocks are still using the `align` attribute for text alignment. Do you think they should be updated to `textAlign` for consistency? Let me know if I should open a new issue for that and submit a PR.

Thank you!

**Comment by jorgefilipecosta** [comment]
From the code point of view, things look and test well to me 👍
@karmatosed would it be possible to have a check on the Design/UX?

**Review by karmatosed — APPROVED** [review]
Thanks for this PR. I am going to approve based on design, looks like we have a code review already so let's get this in!

**Pull request merged**